docs/product-user-guide/src/main/asciidoc/data-sets-caching-con.adoc

```diff
@@ -1,27 +1,30 @@
 [id='data_sets_caching_con']
 = Caching
 
+{CENTRAL} provides caching mechanisms for storing data sets and performing data operations using in-memory data. Caching data reduces network traffic, remote system payload, and processing time. To avoid performance issues, configure the cache settings in {CENTRAL}.
+
+For any data lookup call that result in a data set, the caching technique will determine where the data lookup call is executed and where the resulting data set is stored. An example of a data lookup call would be all the mortgage applications whose locale parameter is set as "Urban".
+
 {CENTRAL} data set functionality provides two cache levels:
 
 * Client level
-* Back-end level
+* Back end level
 
 [float]
 == Client cache
 
-When this cache is turned on, the data set is cached in a web browser during the look-up operation.
-Consequently, further look-up operations do not perform requests to the back end.
+When the cache is turned on, the data set is cached in a web browser during the lookup operation and further lookup operations do not perform requests to the back end. Data set operations like grouping, aggregations, filtering, and sorting are processed in the web browser. Enable client caching only if the data set size is small, for example, for data sets with less than 10 mb size. For large data sets, browser issues such as slow performance or intermittent freezing can occur. Client caching reduces the number of back end requests including requests to the storage system.
 
 [float]
-== Back-end cache
+== Back end cache
 
-When this cache is turned on, the data set is cached by the {CENTRAL} engine.
-Doing this reduces the number of requests to the remote storage system.
+When the cache is enabled, {CENTRAL} engine caches the data set. This reduces the number of requests to the remote storage system. All data set operations are performed in the {CENTRAL} engine using in-memory data. Enable back-end caching only if the data set size is not updated frequently and it can be stored and processed in memory. Using back-end caching is also useful in cases with low latency connectivity issues with the remote storage.
 
 [NOTE]
 ====
 The Java and CSV data providers rely on back-end caching.
 ====
+
 [NOTE]
 ====
 Back-end cache settings are not always visible in the *Advanced* tab of the *Data Set Editor*.
```

Comment:
I know this was not edited as a part of this PR, but could we maybe specify why it is not always visible?